app/models/preneeds/receive_application.rb

```diff
@@ -8,6 +8,7 @@ class ReceiveApplication < Common::Base
     attribute :return_code, Integer
     attribute :application_uuid, String
     attribute :return_description, String
+    attribute :submitted_at, Time, default: ->(_, __) { Time.zone.now }
```

Comment:
What are the usual two arguments to this lambda?

Reply:
the first one is the instance of the model, the 2nd is an instance of `Virtus::Attribute`, not sure what it's used for